 CMakeLists.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 35077f3c2..99eeb4817 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -290,7 +290,7 @@ foreach (OUTPUTCONFIG ${CMAKE_CONFIGURATION_TYPES})
   set_target_properties( ${OpenBLAS_LIBS} PROPERTIES ARCHIVE_OUTPUT_DIRECTORY_${OUTPUTCONFIG} ${PROJECT_BINARY_DIR}/lib/${OUTPUTCONFIG} )
 endforeach()
 
-enable_testing()
+#enable_testing()
 
 if (USE_THREAD)
   # Add threading library to linker
@@ -313,26 +313,26 @@ endif()
 if (NOT NO_CBLAS)
   if (NOT ONLY_CBLAS)
   # Broken without fortran on unix
-    add_subdirectory(utest)
+#    add_subdirectory(utest)
 endif()
 endif()
 
 if (NOT NOFORTRAN)
  if (NOT ONLY_CBLAS)
   # Build test and ctest
-  add_subdirectory(test)
+#  add_subdirectory(test)
  endif()
   if (BUILD_TESTING)
-    add_subdirectory(lapack-netlib/TESTING)
+#    add_subdirectory(lapack-netlib/TESTING)
   endif()
 endif()
   if(NOT NO_CBLAS)
    if (NOT ONLY_CBLAS)
-    add_subdirectory(ctest)
+#    add_subdirectory(ctest)
    endif()
   endif()
   if (CPP_THREAD_SAFETY_TEST OR CPP_THREAD_SAFETY_GEMV)
-    add_subdirectory(cpp_thread_test)
+#    add_subdirectory(cpp_thread_test)
   endif()
 
 set_target_properties(${OpenBLAS_LIBS} PROPERTIES
 