 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3761965..edf72be 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -513,6 +513,8 @@ if(WEBP_BUILD_IMG2WEBP)
                         exampleutil
                         imagedec
                         imageioutil
+                        liblzma
+                        zstd_static
                         webp
                         libwebpmux)
   target_include_directories(img2webp PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/src)
@@ -577,7 +579,7 @@ if(WEBP_BUILD_EXTRAS)
 
   # get_disto
   add_executable(get_disto ${GET_DISTO_SRCS})
-  target_link_libraries(get_disto imagedec)
+  target_link_libraries(get_disto imagedec liblzma zstd_static)
   target_include_directories(get_disto
                              PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
                                      ${CMAKE_CURRENT_BINARY_DIR}/src)
